FileIO/readMeshFromFile.cpp

```diff
@@ -44,8 +44,16 @@ MeshLib::Mesh* readMeshFromFile(const std::string &file_name)
 {
 #ifdef USE_PETSC
 	NodePartitionedMeshReader read_pmesh(PETSC_COMM_WORLD);
+
+	std::string file_name_ext;
+	if (BaseLib::hasFileExtension("msh", file_name))
+	   file_name_ext = ".msh";
+	if (BaseLib::hasFileExtension("vtk", file_name))
+	   file_name_ext = ".vtk";
+	if (BaseLib::hasFileExtension("vtu", file_name))
+	   file_name_ext = ".vtu";
```

Comment:
What shall happen in other cases, if the mesh extension is neither msh, nor vtk, nor vtu?